issuing_using_did.md

```diff
@@ -160,63 +168,33 @@ The above structure will sufficiently allow for the identity be bounded for docu
+TBD
 
-```json
-{
-  "schema": "tradetrust/v1.0",
-  "data": {
-    "issuers": [
-      {
-        "name": "TradeTrust Demo",
-        "revocation": {
-          "type": "REVOCATION_STORE",
-          "address": "0xabcd...1234"
-        }
-      }
-    ]
-  },
-  "privacy": {
-    "obfuscatedData": []
-  },
-  "signature": {
-    "type": "SHA3MerkleProof",
-    "targetHash": "61dc9186345e05cc2ae53dc72af880a3b66e2fa7983feaa6254d1518540de50a",
-    "proof": [],
-    "merkleRoot": "61dc9186345e05cc2ae53dc72af880a3b66e2fa7983feaa6254d1518540de50a"
-  },
-  "proof": {
-    "type": "DidGenericSignature",
-    "proofPurpose": "assertionMethod",
-    "signature": "<signed merkle root>",
-    "did": "did:ethr:0xE6Fe788d8ca214A080b0f6aC7F48480b2AEfa9a6",
-    "didPublicKey": {
-      "id": "did:ethr:0xE6Fe788d8ca214A080b0f6aC7F48480b2AEfa9a6#owner",
-      "type": "Secp256k1VerificationKey2018",
-      "ethereumAddress": "0xe6fe788d8ca214a080b0f6ac7f48480b2aefa9a6",
-      "owner": "did:ethr:0xE6Fe788d8ca214A080b0f6aC7F48480b2AEfa9a6"
-    }
-  }
-}
-```
+### Difference with existing PR
 
-## Implementation Details
+[#91](https://github.com/Open-Attestation/oa-verify/pull/91)
+Instead of supporting a single signing method directly, the suggested approach is to support all the DID signing methods eventually, starting with did:ethr signing method. In addition, the specific public key used to sign the document is kept in the document to allow fo easy searching of matching key from the resolver.
 
-TBD
+[#93](https://github.com/Open-Attestation/oa-verify/pull/93)
+Instead of allowing a custom mix and match, the suggested approach is to bundle both document store issue and revoke method into one, and that one should not run without the other. This prevents an ambiguous case where there is both document store and direct signing proof. This is dangerous when the signing key has not been bound to an identity.
 
-## Non-breaking Upgrade
+[#80](https://github.com/Open-Attestation/oa-verify/pull/80)
+Instead of querying the blockchain for the presence of a transaction and the tx.origin of that transaction, we avoid the use of transaction in the first place. In addition, tx.origin is not a common approach for attesting identity as it prevents further abstraction of identity through identity proxies like DSProxy or multisig wallets. The suggested approach is to directly embed the signed message from a key associated with the DID within the document. This way:
 
-TBD
+- no transactions is needed on the ethereum network
+- we know that the DID
+
+## Extension
+
+### Counterfactually deploying document store as revocation store
+
+This allow a user to "reserve" an address for the revocation store in the future but not perform the action now. The user will still be able to use the address during the document creation phase.
```

Comment:
do you mean forcing the deployment of a revocation store on a specific address, and only deploy when needed ?

Reply:
Yes, you can reserve an address with "CREATE2" operation